ush/config_defaults.yaml

```diff
@@ -51,9 +51,14 @@ user:
   # ACCOUNT:
   # The account under which to submit jobs to the queue.
   #
+  # SERVICE_ACCOUNT:
+  # The account under which to submit non-reservation jobs to the queue.
+  # Defaults to ACCOUNT if not set.
+  #
   #-----------------------------------------------------------------------
   MACHINE: "BIG_COMPUTER"
   ACCOUNT: ""
+  SERVICE_ACCOUNT: '{{ user.ACCOUNT }}'
```

Comment:
This is new. I suppose there are systems where you have to use a different account for HPSS?

Reply:
Thanks, I missed `HPSS_ACCOUNT` which was recently added after this PR is opened. They both default to the regular `ACCOUNT`, but there maybe a usecase for them on some systems as you pointed out.

Reply:
Thanks @MichaelLueken @hu5970! We will keep both of them then.